develop/CICD/Tutorials/CICD_Tutorial_Connector.md

```diff
@@ -0,0 +1,298 @@
+
+   This will perform:
+
+    1. Compilation
+
+    1. Unit Testing
+
+    1. DataMiner Connector Validator
+
+## Step 5: Extend the workflow with CD, automatic deployment
+
+1. Navigate to *Actions*
+
+1. On the left, click your *.NET* workflow.
+
+1. At the top, click *dotnet.yml*.
+
+1. In the new window, click the pencil icon to edit the page.
+
+1. Add a new job called *CD* running on a Windows image with new steps:
+
+    1. Only run when the CI job is completed.
+
+    1. Retrieve the source code.
+
+    1. Install the Packager and Deployer tools.
+
+    1. Run the packager to create a .dmprotocol.
+
+    1. Deploy the .dmprotocol to an accessible Agent directly.
+
+   > [!IMPORTANT]
+   >
+   > - Deployment with a local artifact requires running on a Windows OS. For more details, you can take a look at this [GitHub issue](https://github.com/SkylineCommunications/Skyline.DataMiner.CICD.Tools.DataMinerDeploy/issues/10)
+   > - Deployment with a local artifact requires DataMiner 10.3.0/10.3.2 or higher.
+
+   ```yml
+     CD:
+
+       runs-on: windows-latest
+       needs: CI
+       steps:
+       - uses: actions/checkout@v4
+  
+       - name: Install Package Creation
+         run: dotnet tool install -g Skyline.DataMiner.CICD.Tools.Packager
+       - name: Install DataMiner Deploy
+         run: dotnet tool install -g Skyline.DataMiner.CICD.Tools.DataMinerDeploy
+      
+       - name: Create Protocol Package
+         run: dataminer-package-create dmprotocol "${{ github.workspace }}" --name  "protocol" --output "${{ github.workspace }}\\_PackageResults"
+
+       - name: Direct Agent Deployment
+         run: dataminer-package-deploy from-artifact --path-to-artifact "${{ github.workspace }}\\_PackageResults\\protocol.dmprotocol" --dm-server-location "${{ secrets.SERVER_LOCATION }}" --dm-user "${{ secrets.DATAMINER_USER }}" --dm-password "${{ secrets.DATAMINER_PASSWORD }}"
+   ```
+
+1. Commit your changes.
+
+1. Navigate to *Actions* and check the run of your enhanced pipeline.
+
+   This will perform:
+
+    1. Compilation
+
+    1. Unit testing
+
+    1. DataMiner connector validation
+
+    1. Package creation
+
+    1. Direct Agent deployment
+
+## Step 6: Add your GitHub secrets
+
+1. Navigate to *Settings*.
+
+1. On the left, select *Secrets and variables*.
+
+1. Under *Actions*, add a *New repository secret* for the following secrets:
+
+    1. SERVER_LOCATION
+
+    1. DATAMINER_USER
+
+    1. DATAMINER_PASSWORD
+
+## Step 7: Enjoy the results
+
+1. Navigate to *Actions*.
+
+1. Select the *.NET* workflow on the left.
+
+1. Click *Run workflow*.
+
+You should now see your CI and CD jobs complete successfully.
+
+> [!NOTE]
+> Taking a screenshot of this successful run and sending that to <thunder@skyline.be> or uploading it through the [Dojo tutorials page](https://community.dataminer.services/learning-courses-tutorials/) will grant you DevOps points.
+
+## Advanced Options: CI
```

Comment:
I'm not sure if these last two sections (from this line onwards) really belong in this tutorial, as they don't include something the user should do as part of the tutorial. It might be better to include this on a different page and then add a link to that page here.

Reply:
This is mentioned during the Kata as well, I'd prefer to keep it as it links to more advanced information directly from where they just did the tutorial. The information is available under CI/CD docs as well I think. Though not part of the tutorial, it gives a "what's next?" for the users.